easybuild/easyconfigs/c/compleasm/compleasm-0.2.2-foss-2022b.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'compleasm'
+version = '0.2.2'
+
+homepage = 'https://github.com/huangnengCSU/compleasm'
+description = """Compleasm: a faster and more accurate reimplementation of BUSCO. A genome completeness evaluation tool
+based on miniprot"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+github_account = 'huangnengCSU'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['07772e567b6c058e2a685abbe4ab75d9636bd8aebb013b5348744383e4266766']
+
+dependencies = [
+    ('miniprot', '0.12'),
+    ('HMMER', '3.3.2'),
+    ('SEPP', '4.5.1'),
+    ('SciPy-bundle', '2023.02'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': False}
+sanity_check_paths = {
```

```suggestion
]

sanity_check_paths = {
```

Required for newer EB version